build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts

```diff
@@ -161,14 +166,19 @@ async function updateImports(
 					 * module/path.
 					 */
 					const fullImportSpecifierText = importSpecifier.getFullText().trim();
-					const expectedLevel = getApiLevelForImportName(name, data, "public", onlyInternal);
+					const expectedLevel = getApiLevelForImportName(
+						name,
+						data,
+						/* default */ publicLevel,
```

Comment:
This is a reason to use param object more often. string and booleans and numbers are too often easily mixed up. No fix required.